cypress/integration/settings.spec.ts

```diff
@@ -19,6 +21,117 @@ context("settings", () => {
+      commands.pick("add-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+    });
+
+    it("can add a seed via enter", () => {
+      cy.get(".seeds").find(".seed").should("have.length", 0);
+      commands.pasteInto(["seed-input"], validSeedAddress);
+      commands.pick("seed-input").type("{enter}");
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+    });
+
+    it("persists adding a seed across app start", () => {
+      cy.get(".seeds").find(".seed").should("have.length", 0);
+      commands.pasteInto(["seed-input"], validSeedAddress);
+      commands.pick("add-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+      commands.restartAndUnlock();
+      commands.pick("sidebar", "settings").click();
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+    });
+
+    it("prevents adding the same seed", () => {
+      // add a seed
+      commands.pasteInto(["seed-input"], validSeedAddress);
+      commands.pick("add-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+
+      // add the same seed again
+      commands.pasteInto(["seed-input"], validSeedAddress);
+      commands.pick("add-seed").click();
+      cy.get(".seed-entry-form").contains("This seed already exists");
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+    });
+
+    it("adds new seeds to the end of the list", () => {
+      commands.pasteInto(["seed-input"], `${validSeedAddress}1`);
+      commands.pick("add-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+
+      // add a second seed
+      commands.pasteInto(["seed-input"], `${validSeedAddress}2`);
+      commands.pick("add-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 2);
+      cy.get(".seeds").find(".seed").last().contains(`${validSeedAddress}2`);
+
+      // add a third seed
+      commands.pasteInto(["seed-input"], `${validSeedAddress}3`);
+      commands.pick("add-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 3);
+      cy.get(".seeds").find(".seed").last().contains(`${validSeedAddress}3`);
+    });
+
+    it("can delete a seed and persists the change across app start", () => {
+      // add a seed
+      cy.get(".seeds").find(".seed").should("have.length", 0);
+      commands.pasteInto(["seed-input"], validSeedAddress);
+      commands.pick("add-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 1);
+
+      // remove the seed
+      commands.pick("remove-seed").click();
+      cy.get(".seeds").find(".seed").should("have.length", 0);
+
+      // the change is persisted across app start
+      commands.restartAndUnlock();
+      commands.pick("sidebar", "settings").click();
+      cy.get(".seeds").find(".seed").should("have.length", 0);
+    });
+
+    it("persists the order of the seed list when removing one", () => {
```

Comment:
I think we have to be a conscious about the number of tests and the overall time it takes to run the suite. Cypress test cases take quite some time to set up.

I would suggest merging the following tests into one
* can add a seed via button
* adds new seeds to the end of the list
* can delete a seed and persists the change across app star
* adds new seeds to the end of the list
* persists the order of the seed list when removing one

We would have the same test coverage and the tests run faster. The downside would be that it is harder to isolate simple test cases on failure.